sgkit/tests/test_aggregation.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any
+from typing import Any, List, Union
```

Comment:
The changes in this file are no longer needed, are they?

Reply:
Actually, these were never required from test/mypy point of view since they are ignored for tests,  I did this to fix the type hint for `calls` param of `get_dataset` as we are inputting List of List as an argument. What do you think should we keep it?